doc/manual/syntax.md

```diff
@@ -147,13 +147,13 @@ present on all lines of the string is stripped. This way, multiline strings can
 be indented for nicer code formatting without producing unwanted whitespaces in
 the output. For example:
 
-```nickel
+```nickel-repl
 > m%"
-  This line has no indentation.
-    This line is indented.
-      This line is even more indented.
-  This line has no more indentation.
-"%
+    This line has no indentation.
```

Comment:
I'm a little confused by this example. I guess you had to indent it extra so that the final `"%` will be caught by the parser? But even before this change, the "This line has no indentation" line had indentation...

Reply:
The multiline string parser strips common indentation. So this line doesn't have any indentation in the final string and that's what this example is supposed to illustrate.